tests/functional_tests/0003_create_test.py

```diff
@@ -161,27 +156,46 @@ def test_09_test_dhcp_connectivity_in_jail(release, jail, invoke_cli):
     ip = jail.ip
     assert ip is not None and '0.0.0.0' not in ip
 
+    # Let's test if we can ping 8.8.8.8 from inside the jail
+
+    stdout, stderr = jail.run_command(['ping', '-c', '5', '8.8.8.8'])
+
+    assert bool(stderr) is False, f'ping returned an error: {stderr}'
+
 
 @require_root
 @require_zpool
-@require_networking
-def test_10_create_jail_and_install_packages(release, jail, invoke_cli):
-    # FIXME: Test to ensure the packages were installed
+@require_dhcp
+def test_10_create_jail_and_install_packages(release, jail, invoke_cli, skip_test):
 
+    pkg_jail = 'pkg_jail'
+    install_pkgs = ['nano']
     fd, path = tempfile.mkstemp()
+
     try:
         with os.fdopen(fd, 'w') as tmp:
             tmp.write(json.dumps({
-                'pkgs': ['nginx']
+                'pkgs': install_pkgs
             }))
 
         invoke_cli(
-            ['create', '-r', release, '-n', 'pkg_jail', '-p', path]
+            ['create', '-r', release, '-n', pkg_jail, '-p', path, 'dhcp=on']
```

Comment:
We should allow the pkg test to happen if they don't supply dhcp.

Reply:
As in with a manual ip address ?

Reply:
Networking is required for this test

Reply:
Yeah with a manual IP that they can specify. If they do, and not dhcp then they can't actually test this.